knotx-mocks/src/main/java/io/knotx/mocks/knot/KnotContextKeys.java

```diff
@@ -80,9 +81,10 @@ String key() {
 
   Observable<Optional<Object>> mockValue(FileSystem fileSystem, String resourcePath) {
```

Comment:
We use `Observable` in `handle` method which is just more natural here because `Single` enables emitting only **one** message while in our case we support the situation with no messages (result is 404 status code). Using `Single` here is probably possible, but `Observable` is just more natural.